src/ListItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+interface person {
```

Comment:
These are the attributes that were storing so you might need to add them to the interface to be displayed:

firstName
lastName
avgDifficulty
avgRating
numRatings
wouldTakeAgainPercent

Reply:
id and legacyId are also available but I don't think that these will be shown on front end